content/new-in-22.md

```diff
@@ -169,9 +154,24 @@ Previous changes are listed in the [changelog](https://www.w3.org/TR/WCAG22/#cha
 
 Provide ways to help users navigate, find content, and determine where they are.
 
-### 2.4.11 Focus Not Obscured (Minimum) (AA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#reporter">Reporter</a> with repetitive stress injury who uses speech recognition software:</p>
+### [AG version] 2.4.11 Focus Not Obscured (Minimum) (AA)
+<section class="brief">
+      <p>In brief:</p>
+      <dl>
+         <dt>Objective</dt><dd>Do not hide the item with focus</dd>
+         <dt>Author task</dt><dd>The item with focus should be at least partially visible in the viewport</dd>
+         <dt>Key beneficiaries</dt><dd>Some users with cognitive disabilities and sighted users reliant on keyboard interaction</dd>
+      </dl>
+  </section>
+
+### [EOWG draft] 2.4.11 Focus Not Obscured (Minimum) (AA)
+In brief:
+<ul>
+  <li><p><span class="whatwhy">What to do: </span>Ensure that when items have focus, they are at least partially visible.</p> </li>
+  <li><p><span class="whatwhy">Why: </span>Some people cannot use a mouse and rely on seeing what has keyboard focus.</p> </li>
```

Comment:
From [Rebab](https://www.w3.org/2002/09/wbs/35532/wcag2_2_in_brief/results):
I see a benefit from adding the cognitive disabilities -``Key beneficiaries`` in the why. It resonates more: 
Key beneficiaries: Some users with cognitive disabilities and sighted users reliant on keyboard interaction.
Current why: Some people cannot use a mouse and rely on seeing what has keyboard focus.

Proposed why: Some users with cognitive disabilities and those who don`t use a mouse, rely on seeing the item that has keyboard focus on.

Reply:
My concern with going this route is the same reason I think it made sense to remove the Key beneficiaries information. It becomes so simplified that it becomes almost non-representative. It's why I'm hoping for more detail in the Benefits area.